### ISSUE TITLE
Extend KeyVault functionality 

### ISSUE THREAD

**Issue by cmatskas** [bradygaster/azure-tools-vscode#25]
@bradygaster Apart from creating KVs, we should be able to use AVTS to interact with KV data. </Shameless plug>  most of the code for this work is here: https://github.com/cmatskas/azure-keyvault-node/blob/master/kvservice.js. I could bring that service in and edit to fit our needs.
Suggested operations:
**KV Instance**
- Acquire KV client

**Secret Operations**
- Add secret
- Delete secret
- Get secret

**Key Operations**
- Add key
- Delete key
- Encrypt with key
- Decrypt with key
- Sign with key

**Comment by bradygaster** [comment]
Looks like you've done most of the back-end for this in your code from the post. Adding to the 1.2.0 milestone to follow up on the last set of work. then we'll tackle document db.

**Comment by bradygaster** [comment]
@cmatskas i think this would be great to see pop in an upcoming release. changing the milestone to push it out since it won't make it for 1.2.0. Any ETA on the list of features you outlined in this issue?

**Comment by cmatskas** [comment]
Yes. This is earmarked for this week! It will be integrated as a whole :) Are you ok to do the UI work? I'm not sure how we can make it happen or if at all possible?
